readme.md

```diff
@@ -11,6 +11,12 @@ This is for advanced users. You probably want to use XO directly.
 $ npm install --save-dev eslint-config-xo
 ```
 
+For the `esnext` version you'll also need Babel's eslint parser and plugin:
+
+```
+$ npm install --save-dev babel-eslint eslint-plugin-babel
+```
```

Comment:
Annoying that this is needed. Could use some support on: https://github.com/eslint/eslint/issues/3458